src/core/turn-order.js

```diff
@@ -37,15 +37,53 @@ export const Pass = (G, ctx) => {
  * begins, and also a function `next` to determine who the
  * next player is when the turn ends.
  */
+export function playerOrderPos(ctx) {
+  return ctx.playerOrder.findIndex(
+    nextPlayer => nextPlayer == ctx.currentPlayer
+  );
+}
+
 export const TurnOrder = {
   /**
    * DEFAULT
    *
    * The default round-robin turn order.
    */
   DEFAULT: {
-    first: (G, ctx) => ctx.currentPlayer,
-    next: (G, ctx) => (+ctx.currentPlayer + 1) % ctx.numPlayers + '',
+    first: (G, ctx) => ctx.playerOrder[0] + '',
+    next: (G, ctx) => {
+      return ctx.playerOrder[(playerOrderPos(ctx) + 1) % ctx.numPlayers] + '';
+    },
+  },
+
+  /**
+   * ONETIME
+   *
+   * The default round-robin turn order only one time.
+   */
+  ONETIME: {
+    first: (G, ctx) => ctx.playerOrder[0] + '',
+    next: (G, ctx) => {
+      if (playerOrderPos(ctx) < ctx.numPlayers - 1) {
```

Comment:
What happens when this function returns `undefined` (which happens when this `if` condition is false)?